benchmarks/shootout/test_on_sgx.sh

```diff
@@ -18,32 +18,45 @@ if [[ -z $(which ${WAMR_DIR}/wamr-compiler/build/wamrc) ]]; then
     exit 1
 fi
 
-if [[ -z $(which ${WAMR_DIR}/product-mini/platforms/linux-sgx/enclave-sample/iwasm) ]]; then
+if [[ -z $(which ${WAMR_DIR}/product-mini/platforms/linux-sgx-1/enclave-sample/iwasm) ]]; then
```

Comment:
Could we rename the folder name to `linux-sgx-classic-interp`, `linux-sgx-fast-interp`, `linux-sgx-aot`, and `linux-sgx-fast-jit`? And build and test the four modes?

Reply:
Okay, I tested interp and aot together, I will separate those two. And I will rename them accordingly